customer/src/main/java/no/unit/nva/customer/CustomerDoiHandler.java

```diff
@@ -29,6 +46,18 @@ protected UUID getIdentifier(RequestInfo requestInfo) throws InputException {
                    .orElseThrow(fail -> handleIdentifierParsingError(identifier, fail));
     }
 
+    protected Map<UUID, SecretManagerDoiAgentDao> getSecretsManagerDoiAgent() throws JsonProcessingException {
+
+        var secretAsStringJsonArray = secretsReader.fetchSecret(SECRETS_KEY_AND_NAME, SECRETS_KEY_AND_NAME);
```

Comment:
They use the same constant `SECRETS_KEY_AND_NAME` ?

Reply:
Yes!